platform/node/test/ignores.json

```diff
@@ -39,6 +39,7 @@
   "render-tests/custom-layer-js/null-island": "skip - js specific",
   "render-tests/custom-layer-js/tent-3d": "skip - js specific",
   "render-tests/regressions/mapbox-gl-js#7708": "skip - js specific",
+  "render-tests/debug/overdraw": "requires a debug build",
```

Comment:
Could we add issue for that? So that we exclude this test from release builds.

Reply:
https://github.com/mapbox/mapbox-gl-native/issues/15638